en/guide/mavlink_v1_compatibility_features.md

```diff
@@ -0,0 +1,97 @@
+
+## Message Set Requirements
+
+The message set requirements should be interpreted as follows: If the autopilot sends any dimension contained in the messages below, it has to use these messages and not deprecated other messages to do so. 
+In addition, [HEARTBEAT](../messages/common.md#HEARTBEAT), [SYS_STATUS](../messages/common.md#SYS_STATUS) and [AUTOPILOT_VERSION](../messages/common.md#AUTOPILOT_VERSION) have to be sent repeatedly independent of the system state.
+
+Any message being mandated on the TX side has to be handled correctly on the receiving side (e.g. a message mandated in the autopilot TX side has to be parsed correctly in the GCS RX side, and vice-versa).
+
+## Minimum Message Profile
+
+The minimum message set is intended for barebones operation.
+
+### Autopilot TX
+
+* [HEARTBEAT](../messages/common.md#HEARTBEAT)
+* [SYS_STATUS](../messages/common.md#SYS_STATUS)
+* [AUTOPILOT_VERSION](../messages/common.md#AUTOPILOT_VERSION)
+* [ATTITUDE_QUATERNION](../messages/common.md#ATTITUDE_QUATERNION)
+* [GLOBAL_POSITION_INT](../messages/common.md#GLOBAL_POSITION_INT)
+* [GPS_RAW_INT](../messages/common.md#GPS_RAW_INT)
+* [RC_CHANNELS](../messages/common.md#RC_CHANNELS)
+* [BATTERY_STATUS](../messages/common.md#BATTERY_STATUS)
+* [HIGHRES_IMU](../messages/common.md#HIGHRES_IMU)
+* [STATUSTEXT](../messages/common.md#STATUSTEXT)
+
+### GCS TX
+
+* [HEARTBEAT](../messages/common.md#HEARTBEAT)
+* [MANUAL_CONTROL](../messages/common.md#MANUAL_CONTROL)
+
+
+## Parameter Protocol Profile
+
+In addition to the minimum message profile, the parameter profile requires handling of these messages:
+
+### Autopilot TX
+
+* [PARAM_VALUE](../messages/common.md#PARAM_VALUE)
+
+### GCS TX
+
+* [PARAM_REQUEST_READ](../messages/common.md#PARAM_REQUEST_READ)
+* [PARAM_REQUEST_LIST](../messages/common.md#PARAM_REQUEST_LIST)
+* [PARAM_SET](../messages/common.md#PARAM_SET)
+
+## Mission Protocol Profile
+
+In addition to the minimum message profile, the mission profile requires handling of these messages:
+
+### Autopilot TX
+
+* [MISSION_ITEM](../messages/common.md#MISSION_ITEM)
+* [MISSION_REQUEST_LIST](../messages/common.md#MISSION_REQUEST_LIST)
+* [MISSION_CLEAR_ALL](../messages/common.md#MISSION_CLEAR_ALL)
+* [MISSION_ITEM_REACHED](../messages/common.md#MISSION_ITEM_REACHED)
+* [MISSION_ACK](../messages/common.md#MISSION_ACK)
+
+### GCS TX
+
+* [MISSION_ITEM](../messages/common.md#MISSION_ITEM)
+* [MISSION_ITEM_INT](../messages/common.md#MISSION_ITEM_INT) (preferred)
+* [MISSION_REQUEST](../messages/common.md#MISSION_REQUEST)
+* [MISSION_CURRENT](../messages/common.md#MISSION_CURRENT)
+* [MISSION_COUNT](../messages/common.md#MISSION_COUNT)
+* [MISSION_ACK](../messages/common.md#MISSION_ACK)
+
+## Robotics / Companion Computer Protocol Profile
+
+In addition to the minimum message profile, the robotics profile requires handling of these messages:
```

Comment:
This would only be true for the Autopilot side right? ie a Companion computer would not be required to handle the whole minimum message profile - including AUTOPILOT _VERSION, ATTITUDE_QUATERNION etc. ?